advocacy_docs/partner_docs/CohesityBackupandRecover/04-ConfiguringCohesityConnectorAgentforEDBPostgreSQL.mdx

```diff
@@ -0,0 +1,76 @@
+- Credentials that are used to access the EDB Postgres Advanced Server database
+- Java 1.8 or higher version is installed on the PostgreSQL nodes
+
+### Configure Cohesity for EDB Postgres Advanced Server
+
+**Register and Manage the EDB Postgres Advanced Server Source:**
+
+1. Log on to the Cohesity console at the IP address that was used to create your VMware Cohesity Backup Appliance with the format: `https://ipaddress/login`
+
+   ![Cohesity Login Page](Images/CohesityLoginPage.png)
+
+2. When the Cohesity Dashboard login page appears, login with your Cohesity credentials.
+
+   ![Cohesity Login Credentials](Images/CohesityLoginCredentials.png)
+
+3. Once logged in, you will then be presented with the Cohesity Dashboard.
+
+   ![Cohesity Dashboard](Images/CohesityDashboard.png)
+
+4. Register your EDB Postgres Advanced Server database. From the main Dashboard screen select Data Protection and then select Sources on the left-hand side.
+
+
+5. On the Sources page, click on the `Register` button on the top right side and click on the `Universal Data Adapter` option.
+
+   ![Cohesity Source Registration](Images/CohesityRegisterSource.png)
+
+
+6. Step through the Universal Data Adapter screens by clicking `Next` until you get to the Register Universal Data Adapter screen:
+
+   ![Cohesity Universal Data Adapter](Images/CohesityUniversalDataAdapter.png)
+
+   ![Cohesity Register Universal Data Adapter](Images/CohesityRegisterUniversalDataAdapter.png)
+
+7. Once on the Register Universal Data Adapter screen input the specific source information in order for the source to be protected.
+- **Source Type**: For EDB Postgres Advanced Server, Source Type will be PostgreSQL
+- **Hostnames/IP Addresses**: This will be either the Hostname or IP Address of the specific source to register
+- **Script Directory Path**: This path is defined during the installation of the Cohesity Postgres Agent.  It will be displayed at the bottom of the installation scripts when the download is complete and it is the location of where the scripts are located: /opt/cohesity/postgres/scripts
+- In the **App Authentication** section enter the admin username and password for the user who has admin privileges on your database to perform a backup.
+      !!! Note- Instead of password-based authentication, if you want to use kerberos authentication, then leave the username and password fields blank.
+- Select `Custom Options` and proceed with the information below for your EDB Postgres Advanced Server source.
+
+
+   ![Cohesity Universal Data Adapter Information](Images/CohesityUniversalDataAdapterInformation.png)
+
+8. Select `Custom Options`
+
+   ![Cohesity Global Source Registration](Images/CohesityGlobalSourceRegistration.png)
+
+9. In the Global Source Registration Arguments field enter the following with the same syntax:
+--source-name=cohesitypostgres --postgres-user=enterprisedb --host=172.24.113.13 --port=5444 --pg-bin=/usr/edb/as14/bin
```

Comment:
Use code syntax for "--source-name=cohesitypostgres --postgres-user=enterprisedb --host=172.24.113.13 --port=5444 --pg-bin=/usr/edb/as14/bin"